svalinn.yaml

```
@@ -46,10 +46,9 @@ webhook:
   timeout: "1m"
   registrationURL: "https://127.0.0.1:6000/hook"
   eventsToWatch: ["device-status.*"]
-  acquirerConfig:
-    basic: 
-    sat:
-      client: ""
-      secret: ""
-      satURL: ""
-      timeout: "1m"
\ No newline at end of file
+  basic: 
```

```suggestion
  # basic: 
```
Since only one will work comment out the other one?

---

I just put them both as options, as they both are in the config file right now, neither will be chosen and it will be the default acquirer.  If you want basic to be commented out, then sat should be commented out too.

---

makes sense